openedx/core/djangoapps/user_api/views.py

```diff
@@ -669,26 +669,58 @@ def _add_terms_of_service_field(self, form_desc, required=True):
         terms_text = _(u"Terms of Service")
         terms_link = u"<a href=\"{url}\">{terms_text}</a>".format(
             url=marketing_link("TOS"),
-            terms_text=terms_text
+            terms_text=terms_text,
         )
 
-        # Translators: "Terms of service" is a legal document users must agree to
```

Comment:
Wait, I thought you and @caesar2164 had said there was a simple feature flag to support this;
was I mistaken?